source/speakers/index.html.erb

```diff
@@ -0,0 +1,40 @@
+---
+published_at: 2020-11-18 00:00:00 Z
+title: Speakers
+created_at: 2020-11-18 00:00:00 Z
+updated_at: 2020-11-18 00:00:00 Z
+status: Published
+created_by:
+  email: me@mikerogers.io
+  name: Mike Rogers
+---
+
+<div class="speaker__list">
+
+  <% all_speakers.each do |speaker| %>
+    <div class="speaker">
+      <h2 class="speaker__name"><%= speaker.name %></h2>
+      <div class="speaker__socials">
+
+        <% speaker.links.each do |speaker_link| %>
+          <% link_to speaker_link do %>
+            <% if speaker_link.include?('twitter.com') %>
+              <%= image_tag 'twitter.svg', width: 24, height: 24 %>
+            <% elsif speaker_link.include?('github.com') %>
+              <%= image_tag 'github.svg', width: 24, height: 24 %>
+            <% else %>
+              🔗
```

Comment:
Can we get an SVG for a standard link svg too?  I don't know where the twitter and GitHub ones came from, but the link emoji looks different and has a different baseline, so hopefully we can get a link svg from the same place.